integration-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Start a SPIRE Server and Agent and run the integration tests
+# Only works on Linux.
+
+set -euf -o pipefail
+
+spire_folder="spire-858d04b"
+
+function cleanup() {
+  killall -9 spire-agent || true
+  killall -9 spire-server || true
+  rm -f /tmp/spire-server/private/api.sock
+  rm -f /tmp/spire-agent/public/api.sock
+  rm -rf ${spire_folder}
+}
+
+# Some cleanup: kill spire processes that could have remained from previous run
+cleanup
+
+# Install and run a SPIRE server
+curl -s -N -L https://github.com/spiffe/spire/releases/download/v1.0.0/spire-1.0.0-linux-x86_64-glibc.tar.gz | tar xz
+pushd "${spire_folder}"
+bin/spire-server run -config conf/server/server.conf &
```

Comment:
I think it would be clearer to redirect `stdout` and `stderr` to a log file so that the log output from the Server + Agent isn't interleaved with the script/test output.

I put an example below, ideally the directory should extracted into a variable as well
```suggestion
mkdir -p /tmp/spire-server
bin/spire-server run -config conf/server/server.conf >/tmp/spire-server/spire-server.log 2>&1 &
```